test/run_tests

```diff
@@ -48,7 +40,7 @@ parser.add_argument(
     "-d", "--directory",
     type=str, required=False,
     default=os.path.abspath(os.path.dirname(__file__)),
-    help="The test directory to process"
+    help="The directory to search for test files"
 )
 
 parser.add_argument(
```

Comment:
I've always not liked the `-t` option requiring `-d` and then taking the filename for the `-t` argument. Rather, `-t` accepting a path would be much nicer (tab complete). Is that change too big to add to this PR? I'm not sure how this would work for the C5G7 tests that are distinguished by a command line argument, but that problem pre-dated this PR.

Reply:
Let me see what that would take to fix. For something like C5G7, if you specify the test file using `-t`, it will run the file with all of the lua argument options.

Reply:
Fixed. `--test/-t` now takes the full path to a test file and doesn't require `--directory/-d`. Tab completion should work as well as the shell will take care of that.